webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -105,15 +105,13 @@ export class TasksComponent implements OnInit, OnDestroy {
     this.debouncedReload = _debounce(this.refreshTasks.bind(this), 1000, { maxWait: 10 * 1000 });
 
     this.currentLevel = this.sessionService.isOnlineOnly() ? Promise.resolve() : this.getCurrentLineageLevel();
-
     this.refreshTasks();
   }
 
   ngOnDestroy() {
     this.subscription.unsubscribe();
 
     this.tasksActions.setTasksList([]);
-    this.tasksActions.setTasksLoaded(false);
```

Comment:
Why remove this? Is it because we keep tasks in the Store and not refreshing?

Reply:
That was part of my understanding of the fix. If we setTasksLoaded as false on `ngOnDestroy()` it means when you navigate to a different tab then back to the Tasks Tab, you will always be logging `tasks:load` everytime

Reply:
Yes, that's okay; it's the same behavior for the contacts, reports, and messages tab. Whenever the chw navigates to any of those tabs, it will record an Apdex; because the background processes can affect the experience every time the chw visits the tabs.